utils/aws-operations.js

```diff
@@ -0,0 +1,93 @@
+const AWS = require('aws-sdk');
+const config = require('config');
+
+const {
+  accessKeyId,
+  secretAccessKey,
+  region,
+  endpoint,
+  s3ForcePathStyle,
+} = config.get('aws');
```

Comment:
I don't think you need to destructure them. Instead, I believe you can do something like:

```js
const awsConfig = config.get('aws');
const s3 = new AWS.s3(awsConfig);
```